config/routes.rb

```diff
@@ -121,8 +121,11 @@
 
   # -------------------------------------------------------------- Static pages
   # jQuery Textile URLs
-  post '/preview' => 'home#textilize',  as: :preview
-  get '/markup-help' => 'home#markup_help', as: :markup
+  get '/field', to: 'textile#field', as: :field, defaults: { format: :js }
```

Comment:
Isn't there a RESTful way to approach this other than 5 different individual entries?